chart/samples/istio-sample-revisionbased.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.23.2
+  namespace: istio-system
+  updateStrategy:
+    type: RevisionBased
+---
+apiVersion: sailoperator.io/v1alpha1
```

Comment:
@dgn JFYI, I am seeing the following reconcile error in the Sail Operator logs when applying this yaml.

```
sail-operator 2024-10-29T12:40:01Z    ERROR    ctrlr.revtag    Reconciler error    {"IstioRevisionTag": "default", "reconcileID": "1ff96be2-388e-41e6-a437-fed29156cc84", "error": "failed to retrieve IstioRevision for IstioRevisionTag \"default\": referenced Istio has no active revision"}
sail-operator sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller[...]).reconcileHandler
sail-operator     sigs.k8s.io/controller-runtime@v0.19.0/pkg/internal/controller/controller.go:316
sail-operator sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller[...]).processNextWorkItem
sail-operator     sigs.k8s.io/controller-runtime@v0.19.0/pkg/internal/controller/controller.go:263
sail-operator sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller[...]).Start.func2.2
sail-operator     sigs.k8s.io/controller-runtime@v0.19.0/pkg/internal/controller/controller.go:224
sail-operator 2024-10-29T12:40:01Z    INFO    ctrlr.istio    Reconciling    {"Istio": "default", "reconcileID": "0580456b-b418-4041-a6f8-cd15d7d8d6a3"}
```

Reply:
Yeah that is a problem. I'll check if I can make it look less like an error

Reply:
should be fixed now!